src/js/user-profile/components/AuthApplicationSection.jsx

```diff
@@ -1,18 +1,26 @@
+import PropTypes from 'prop-types';
 import React from 'react';
-import { connect } from 'react-redux';
-
-import { handleVerify } from '../../common/helpers/login-helpers.js';
 
 class AuthApplicationSection extends React.Component {
   constructor(props) {
     super(props);
-    this.handleVerify = handleVerify;
+    this.handleVerify = this.handleVerify.bind(this);
+  }
+
+  handleVerify() {
```

Comment:
Yep, sounds like what existed in `login-helpers.js` for `handleVerify` isn't how we want to handle this going forward, but the new `handleVerify` functions seem very similar in both places, so we could possibly put that back into `login-helpers.js`.